# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg-server/AllowEmptyInput.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xorg-server-1.5.3/hw/xfree86/common/xf86Config.c.vanilla	2009-01-06 13:44:51.000000000 +0000
+++ xorg-server-1.5.3/hw/xfree86/common/xf86Config.c	2009-01-06 13:55:12.000000000 +0000
@@ -1089,7 +1089,7 @@
 #endif
 
     /* AllowEmptyInput is automatically true if we're hotplugging */
-    xf86Info.allowEmptyInput = (xf86Info.autoAddDevices && xf86Info.autoEnableDevices);
+    xf86Info.allowEmptyInput = FALSE; /* (xf86Info.autoAddDevices && xf86Info.autoEnableDevices); */
     xf86GetOptValBool(FlagOptions, FLAG_ALLOW_EMPTY_INPUT, &xf86Info.allowEmptyInput);
 
     xf86Info.useDefaultFontPath = TRUE;
